src/System.Runtime.Numerics/src/System/Numerics/BigIntegerCalculator.AddSub.cs

```diff
@@ -5,6 +5,68 @@ namespace System.Numerics
 {
     internal static partial class BigIntegerCalculator
     {
+        public static uint[] Add(uint left, uint right)
+        {
+            // Executes the trivial addition for two 32-bit integers.
+            // Since the caller is working with uint[] objects, we're
+            // doing him a favor with already providing it that way.
+
+            uint[] bits = new uint[2];
```

Comment:
I'm not following why there'd be no benefit.  There's an array being allocated here, and it's possible bigger than it needs to be.  In general, reducing both the number of allocations and the amount of space allocated leads to better performing applications, as the impact of GC is reduced.  Now, it's possible in this situation the cost of the extra check/branch to determine that a smaller array could be used would outweigh the benefit of allocating less, but I've not seen such an analysis,